linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/EnvTenantHostIdentifierConfig.scala

```diff
@@ -18,10 +18,11 @@ object EnvTenantHostIdentifierConfig {
 }
 
 class EnvTenantHostIdentifierConfig extends HttpIdentifierConfig {
+  val defaultTenant: Option[String] = None
```

Comment:
Could this be moved to line 26?

Reply:
No, this is an attribute of the class. Line 26 would be an argument in the newIdentifier method, which we override.